USAGE.rst

```diff
@@ -53,7 +53,7 @@ And then run the following command for multiple times with different parameter:
 
      python execute_pipeline.py \
        --pipeline_kwargs={\'sleep_secs\':2} \
-       --remote_mode SamplePipeline
```

Comment:
This was changed when the remote pipeline was added, see here:
https://github.com/googleapis/artman/commit/d47a1fd3ee2494744d4687fa00114a342e110605#diff-6e082fcdafbb0217b33680513e81cefaR114